src/ir/value.h

```diff
@@ -107,6 +116,11 @@ class Any {
  public:
   std::string ToString(const SsaNames& ssa_names) const { return "<<ANY>>"; }
 
+  template <typename H>
+  friend H AbslHashValue(H h, const Any& v) {
```

Comment:
We will need to add tests for the hash values.  See https://abseil.io/docs/cpp/guides/hash for how to add tests.

Reply:
Moving Value changes and discussions to another PR

Reply:
Because I had time before the another round of review, added test for hash values.